scripts/generate_openapi.py

```
@@ -0,0 +1,150 @@
+  frontend/src/api/openapi.json
+
+Canonical entrypoint: app.main:app (bootstrap + metrics applied).
+This file must be the single source of truth for OpenAPI generation in CI and locally.
+"""
+
+from __future__ import annotations
+
+import json
+import os
+from pathlib import Path
+from typing import Any, cast, Union
```

Flake8 (7.3.0)

[error] 'typing.cast' imported but unused

(F401)